datalad/distribution/uninstall.py

```diff
@@ -171,7 +180,7 @@ def __call__(
             raise ValueError("`remove_history` flag, requires `remove_handles` flag")
 
         if not remove_data and not remove_handles:
-            raise ValueError("instructured to neither drop data, nor remove handles: cannot perform")
+            raise ValueError("instructed to neither drop data, nor remove handles: cannot perform")
```

Comment:
This conflicts with your description of kill, reading "without consideration of being a dataset, a file, or a directory or any other option given to uninstall"